package.json

```diff
@@ -158,7 +158,7 @@
     "babel-jest": "^23.4.2",
     "babel-loader": "^8.2.1",
     "css-loader": "0.28.11",
-    "cypress": "^6.6.0",
+    "cypress": "^6.9.1",
```

Comment:
did this change generate a new yarn.lock? we should commit that too if it did.

Reply:
The yarn.lock was untouched. I'll need guidance if there's any action that needs to be done.

Reply:
@RachelElysia if you run `yarn install` it should update the yarn.lock. I'm guessing this was updated via Cypress' update button in the UI which is maybe using `npm` rather than `yarn`?

Reply:
Yes, so I have it on yarn now too! Thanks guys.